CMakeLists.txt

```diff
@@ -117,6 +117,10 @@ install(
     DESTINATION "${CMAKE_INSTALL_DATADIR}/everest"
 )
 
+if(BELAY_BOX)
+    install(CODE "execute_process(COMMAND /usr/bin/rsync -avh ${CMAKE_INSTALL_PREFIX}/ everest@${BELAY_BOX}:/mnt/user_data/opt/everest)")
```

Comment:
Is there any ordering in cmake concerning the install steps? Not that this command gets run somewhere in the middle and afterwards files get installed ...

Reply:
I also would prefer to separate these kind of tasks from cmake - but for now, we don't have anything else